### PR TITLE
Fix #907: Faq content and Highfi [ Blocked on #1022]

### LINKED ISSUE #907: Highfi and content for FAQSingleActivity
Mock: https://xd.adobe.com/view/0687f00c-695b-437a-79a6-688e7f4f7552-70b6/screen/195a7967-e7f0-4f62-a99c-8fac560e41d8/HL-FAQ-How-can-I-delete-v2-/

Data: https://docs.google.com/spreadsheets/d/1O9B8oZSXAS3RUm8w2JxHmWRHsh4bNHMOYSRCYLRFK84/edit#gid=0

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

Fixes #907 

This PR finishes the implementation of FAQSingleActivity. This PR includes landscape, content as well as highfi.

Mock: https://xd.adobe.com/view/0687f00c-695b-437a-79a6-688e7f4f7552-70b6/screen/195a7967-e7f0-4f62-a99c-8fac560e41d8/HL-FAQ-How-can-I-delete-v2-/

Landscape mock is not correct and is under discussion for revision so for now this should work.

Data: https://docs.google.com/spreadsheets/d/1O9B8oZSXAS3RUm8w2JxHmWRHsh4bNHMOYSRCYLRFK84/edit#gid=0

## Screenshot:
Screenshot_1585548393
Screenshot_1585548397

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/values/faqs.xml [hunk 1]
```diff
@@ -3,10 +3,19 @@
...
...u003eSet up Multiple Profile\u003c/strong\u003es.\u003cbr\u003e\n2. Create a PIN and \u003cstrong\u003eSave\u003c/strong\u003e.\u003cbr\u003e\n3. Fill in all fields for the profile.\u003cbr\u003e\n\u0026nbsp; a. (Optional) Upload a photo.\u003cbr\u003e\n\u0026nbsp; b. Enter a name.\u003cbr\u003e\n\u0026nbsp; c. (Optional) Assign a 3-digit PIN.\u003cbr\u003e\n4. Tap \u003cstrong\u003eCreate\u003c/strong\u003e. This profile is added to your Profile Chooser!\u003cbr\u003e\n\u003cbr\u003e\nIf you have created a profile before and have a PIN:\u003cbr\u003e\n1. From the Profile Chooser, tap on \u003cstrong\u003eAdd Profile\u003c/strong\u003e.\u003cbr\u003e\n2. Enter your PIN and tap \u003cstrong\u003eSubmit\u003c/strong\u003e.\u003cbr\u003e\n3. Fill in all fields for the profile.\u003cbr\u003e\n\u0026nbsp; a. (Optional) Upload a photo.\u003cbr\u003e\n\u0026nbsp; b. Enter a name.\u003cbr\u003e\n\u0026nbsp; c. (Optional) Assign a 3-digit PIN.\u003cbr\u003e\n4. Tap \u003cstrong\u003eCreate\u003c/strong\u003e. This profile is added to your Profile Chooser!\u003cbr\u003e\n\u003cbr\u003e\nNote: Only the \u003cu\u003eAdministrator\u003c/u\u003e is able to manage profiles.\u003c/p\u003e</string>
```
Comment (nikitamarysolomanpvt) [inline app/src/main/res/values/faqs.xml:12]: Please correct the boldness in "Set up Multiple Profiles."

File: app/src/main/res/values/faqs.xml [hunk 2]
```diff
@@ -3,10 +3,19 @@
...
+  <string name="faq_answer_3">\u003cp\u003eTo change your email/phone number:\u003cbr\u003e\n1. From the Administrator\'s Home Page, tap on the menu button on the top left.\u003cbr\u003e\n2. Tap on \u003cstrong\u003eAdministrator Controls\u003c/strong\u003e.\u003cbr\u003e\n3. Tap on \u003cstrong\u003eEdit Account\u003c/strong\u003e.\u003cbr\u003e\n\u003cbr\u003e\nIf you want to change your email:\u003cbr\u003e\n4. Enter your new email and tap \u003cstrong\u003eSave\u003c/strong\u003e.\u003cbr\u003e\n5. A confirmation link is sent to confirm your new email. The link will expire after __ days and must be clicked on to be associated with your account.\u003cbr\u003e\n\u003cbr\u003e\nIf changing your phone number:\u003cbr\u003e\n4. Enter your new phone number and tap \u003cstrong\u003eVerify\u003c/strong\u003e.\u003cbr\u003e\n5. A code is sent to confirm your new number. The code will expire after __ minutes and must be entered in the new screen to be associated with your account.\u003c/p\u003e</string>
```
Comment (nikitamarysolomanpvt) [inline app/src/main/res/values/faqs.xml:14]: This '__ days' should be like that or should be replaced by some value?

Comment (nikitamarysolomanpvt) [inline app/src/main/res/values/faqs.xml:14]: ditto about '__minutes'..

File: app/src/main/res/values/faqs.xml [hunk 3]
```diff
@@ -3,10 +3,19 @@
...
+  <string name="faq_answer_2">\u003cp\u003eOnce a profile is deleted:\u003cbr\u003e\n- The profile cannot be recovered.\u003cbr\u003e\n- Profile information such as name, photos, and progress will be permanently deleted.\u003cbr\u003e\n\u003cbr\u003e\nTo delete a profile (excluding the \u003cu\u003eAdministrator\'s\u003c/u\u003e):\u003cbr\u003e\n1. From the Administrator\'s Home Page, tap on the menu button on the top left.\u003cbr\u003e\n2. Tap on \u003cstrong\u003eAdministrator Controls\u003c/strong\u003e.\u003cbr\u003e\n3. Tap on \u003cstrong\u003eEdit Profiles\u003c/strong\u003e.\u003cbr\u003e\n4. Tap on the Profile you would like to delete.\u003cbr\u003e\n5. At the bottom of the screen, tap \u003cstrong\u003eProfile Deletion\u003c/strong\u003e.\u003cbr\u003e\n6. Tap \u003cstrong\u003eDelete\u003c/strong\u003e to confirm deletion.\u003cbr\u003e\n\u003cbr\u003e\nNote: Only the \u003cu\u003eAdministrator\u003c/u\u003e is able to manage profiles.\u003c/p\u003e</string>
```
Comment (nikitamarysolomanpvt) [inline app/src/main/res/values/faqs.xml:13]: In mock its showing  "- The profile cannot be recovered." as 
- The profile cannot be recovered. also please compare with mock the other bulleted  points..

**Review by nikitamarysolomanpvt — CHANGES_REQUESTED** [review]
Please fix all suggestions also have a concern on 

The spacing between the question and the first line of answer is comparatively less than that of the spacing after the first line of the answer... which looks little odd..

**Comment by rt4914** [comment]
This issue will be again re-worked once #1022 is finished and merged in `develop`.

**Comment by anandwana001** [comment]
@rt4914 

```
<?xml version="1.0" encoding="utf-8"?>
<resources>
  <string name="faq_question_1">How can I create a new profile?</string>
  <string name="faq_question_2">How can I delete a profile?</string>
  <string name="faq_question_3">How can I change my email/phone number?</string>
  <string name="faq_question_4">Who is an Administrator?</string>
  <string name="faq_question_5">Why is the Exploration player not loading?</string>
  <string name="faq_question_6">Why is my audio not playing?</string>
  <string name="faq_question_7">How do I download a Topic?</string>
  <string name="faq_question_8">I can\'t find my question here. What now?</string>
  <string name="faq_question_9">What is Oppia?</string>
  <string name="faq_answer_1"><![CDATA[<p>If it is your first time creating a profile and not have a PIN: </p> <p> 1. From the Profile Chooser, tap on <strong>Set up Multiple Profiles</strong>. </p> <p> 2. Create a PIN and <strong>Save</strong>. </p> <p> 3. Fill in all fields for the profile. </p> <ol> <li> (Optional) Upload a photo. </li> <li> Enter a name. </li> <li> (Optional) Assign a 3-digit PIN. </li> </ol> <p> 4. Tap <strong>Create</strong>. This profile is added to your Profile Chooser! <br/> <br/> If you have created a profile before and have a PIN: </p> <p> 1. From the Profile Chooser, tap on <strong>Add Profile</strong>. </p> <p> 2. Enter your PIN and tap <strong>Submit</strong>. </p> <p> 3. Fill in all fields for the profile. </p> <ol> <li> (Optional) Upload a photo. </li> <li> Enter a name. </li> <li> (Optional) Assign a 3-digit PIN. </li> </ol> <p> 4. Tap <strong>Create</strong>. This profile is added to your Profile Chooser! <br/> <br/> Note: Only the Administrator is able to manage profiles.</p>]]></string>
  <string name="faq_answer_2"><![CDATA[<p>Once a profile is deleted:</p> <p><br></p> <p>- The profile cannot be recovered.</p> <p>- Profile information such as name, photos, and progress will be permanently deleted.</p> <p><br></p> <p>To delete a profile (excluding the Administrator\'s):</p> <p>1. From the Administrator\'s Home Page, tap on the menu button on the top left.</p> <p>2. Tap on <strong>Administrator Controls</strong>.</p> <p>3. Tap on <strong>Edit Profiles</strong>.</p> <p>4. Tap on the Profile you would like to delete.</p> <p>5. At the bottom of the screen, tap <strong>Profile Deletion</strong>.</p> <p>6. Tap <strong>Delete</strong> to confirm deletion.</p> <p>Note: Only the Administrator is able to manage profiles.</p>]]></string>
  <string name="faq_answer_3"><![CDATA[<p>To change your email/phone number:</p> <p>1. From the Administrator\'s Home Page, tap on the menu button on the top left.</p> <p>2. Tap on <strong>Administrator Controls</strong>.</p> <p>3. Tap on <strong>Edit Account</strong>.</p> <p><br></p> <p>If you want to change your email:</p> <p>4. Enter your new email and tap <strong>Save</strong>.</p> <p>5. A confirmation link is sent to confirm your new email. The link will expire after days and must be clicked on to be associated with your account. </p> <p><br></p> <p>If changing your phone number:</p> <p>4. Enter your new phone number and tap <strong>Verify</strong>.</p> <p>5. A code is sent to confirm your new number. The code will expire after minutes and must be entered in the new screen to be associated with your account.</p>]]></string>
  <string name="faq_answer_4"><![CDATA[<p>An Administrator is the main user that manages profiles and settings for every profile on their account. They are most likely your parent, teacher, or guardian that created this profile for you. </p><p><br></p><p>Administrators have the ability to manage profiles, assign PINs, and change other settings under their account. Depending on your profile, Administrator permissions may be required for certain features such as downloading Topics, changing your PIN, and more. </p><p><br></p><p>To see who your Administrator is, go to the Profile Chooser. The first profile listed and has "Administrator" written under their name is the Administrator. </p>]]></string>
  <string name="faq_answer_5"><![CDATA[<p>If the Exploration Player is not loading</p><p><br></p><p>Check to see if the app is up to date:</p><p>- Go to the Play Store and make sure the app is updated to its latest version</p><p><br></p><p>Check your internet connection:</p><p>- If your internet connection is slow, try re-connecting to your Wi-Fi network or connecting to a different network.</p><p><br></p><p>Ask the Administrator to check their device and internet connection:</p><p>- Get the Administrator to troubleshoot using the steps above</p><p><br></p><p>Let us know if you still have issues with loading:</p><p>- Report a problem by contacting us at admin@oppia.org.</p>]]></string>
  <string name="faq_answer_6"><![CDATA[<p>If your audio is not playing</p><p><br></p><p>Check to see if the app is up to date:</p><p>- Go to the Play Store and make sure the app is updated to its latest version</p><p><br></p><p>Check your internet connection:</p><p>- If your internet connection is slow, try re-connecting to your Wi-Fi network or connecting to a different network. Slow internet may cause the audio to load irregularly, making it difficult to play. </p><p><br></p><p>Ask the Administrator to check their device and internet connection:</p><p>- Get the Administrator to troubleshoot using the steps above</p><p><br></p><p>Let us know if you still have issues with loading:</p><p>- Report a problem by contacting us at admin@oppia.org.</p>]]></string>
  <string name="faq_answer_7"><![CDATA[<p>To download an Exploration:</p><p>1. From the Home Page, tap on a Topic or Exploration.</p><p>2. From that Topic Page, tap on the Info tab.</p><p>3. Tap on Download Topic. </p><p>4. Depending on your app settings, you may need Administrator approval or stable Wifi connection to complete your download. If needed, once these requirements are satisfied, the Topic has been downloaded onto the device and can be used offline by all profiles. <p>]]></string>
  <string name="faq_answer_8"><![CDATA[<p>If you cannot find your question or would like to report a bug, contact us at admin@oppia.org.</p>]]></string>
  <string name="faq_answer_9"><![CDATA[<p>Oppia <i>"O-pee-yah"</i> (Finnish) - "to learn"</p><p><br></p><p>Oppia\'s mission is to help anyone learn anything they want in an effective and enjoyable way.</p><p><br></p><p>By creating a set of free, high-quality, demonstrably effective lessons with the help of educators from around the world, Oppia aims to provide students with quality education — regardless of where they are or what traditional resources they have access to.</p><p><br></p><p>As a student, you can begin your learning adventure by browsing the topics listed on the Home Page!</p>]]></string>
</resources>
```

```
<?xml version="1.0" encoding="utf-8"?>
<resources>
  <!-- FAQ Questions Array -->
  <string-array name="faq_questions">
    <item>@string/faq_question_1</item>
    <item>@string/faq_question_2</item>
    <item>@string/faq_question_3</item>
    <item>@string/faq_question_4</item>
    <item>@string/faq_question_5</item>
    <item>@string/faq_question_6</item>
    <item>@string/faq_question_7</item>
    <item>@string/faq_question_8</item>
    <item>@string/faq_question_9</item>
  </string-array>

  <string-array name="faq_answers">
    <item>@string/faq_answer_1</item>
    <item>@string/faq_answer_2</item>
    <item>@string/faq_answer_3</item>
    <item>@string/faq_answer_4</item>
    <item>@string/faq_answer_5</item>
    <item>@string/faq_answer_6</item>
    <item>@string/faq_answer_7</item>
    <item>@string/faq_answer_8</item>
    <item>@string/faq_answer_9</item>
  </string-array>
</resources>
```

**Comment by rt4914** [comment]
> @rt4914
> 
> ```
> <?xml version="1.0" encoding="utf-8"?>
> <resources>
>   <string name="faq_question_1">How can I create a new profile?</string>
>   <string name="faq_question_2">How can I delete a profile?</string>
>   <string name="faq_question_3">How can I change my email/phone number?</string>
>   <string name="faq_question_4">Who is an Administrator?</string>
>   <string name="faq_question_5">Why is the Exploration player not loading?</string>
>   <string name="faq_question_6">Why is my audio not playing?</string>
>   <string name="faq_question_7">How do I download a Topic?</string>
>   <string name="faq_question_8">I can\'t find my question here. What now?</string>
>   <string name="faq_question_9">What is Oppia?</string>
>   <string name="faq_answer_1"><![CDATA[<p>If it is your first time creating a profile and not have a PIN: </p> <p> 1. From the Profile Chooser, tap on <strong>Set up Multiple Profiles</strong>. </p> <p> 2. Create a PIN and <strong>Save</strong>. </p> <p> 3. Fill in all fields for the profile. </p> <ol> <li> (Optional) Upload a photo. </li> <li> Enter a name. </li> <li> (Optional) Assign a 3-digit PIN. </li> </ol> <p> 4. Tap <strong>Create</strong>. This profile is added to your Profile Chooser! <br/> <br/> If you have created a profile before and have a PIN: </p> <p> 1. From the Profile Chooser, tap on <strong>Add Profile</strong>. </p> <p> 2. Enter your PIN and tap <strong>Submit</strong>. </p> <p> 3. Fill in all fields for the profile. </p> <ol> <li> (Optional) Upload a photo. </li> <li> Enter a name. </li> <li> (Optional) Assign a 3-digit PIN. </li> </ol> <p> 4. Tap <strong>Create</strong>. This profile is added to your Profile Chooser! <br/> <br/> Note: Only the Administrator is able to manage profiles.</p>]]></string>
>   <string name="faq_answer_2"><![CDATA[<p>Once a profile is deleted:</p> <p><br></p> <p>- The profile cannot be recovered.</p> <p>- Profile information such as name, photos, and progress will be permanently deleted.</p> <p><br></p> <p>To delete a profile (excluding the Administrator\'s):</p> <p>1. From the Administrator\'s Home Page, tap on the menu button on the top left.</p> <p>2. Tap on <strong>Administrator Controls</strong>.</p> <p>3. Tap on <strong>Edit Profiles</strong>.</p> <p>4. Tap on the Profile you would like to delete.</p> <p>5. At the bottom of the screen, tap <strong>Profile Deletion</strong>.</p> <p>6. Tap <strong>Delete</strong> to confirm deletion.</p> <p>Note: Only the Administrator is able to manage profiles.</p>]]></string>
>   <string name="faq_answer_3"><![CDATA[<p>To change your email/phone number:</p> <p>1. From the Administrator\'s Home Page, tap on the menu button on the top left.</p> <p>2. Tap on <strong>Administrator Controls</strong>.</p> <p>3. Tap on <strong>Edit Account</strong>.</p> <p><br></p> <p>If you want to change your email:</p> <p>4. Enter your new email and tap <strong>Save</strong>.</p> <p>5. A confirmation link is sent to confirm your new email. The link will expire after days and must be clicked on to be associated with your account. </p> <p><br></p> <p>If changing your phone number:</p> <p>4. Enter your new phone number and tap <strong>Verify</strong>.</p> <p>5. A code is sent to confirm your new number. The code will expire after minutes and must be entered in the new screen to be associated with your account.</p>]]></string>
>   <string name="faq_answer_4"><![CDATA[<p>An Administrator is the main user that manages profiles and settings for every profile on their account. They are most likely your parent, teacher, or guardian that created this profile for you. </p><p><br></p><p>Administrators have the ability to manage profiles, assign PINs, and change other settings under their account. Depending on your profile, Administrator permissions may be required for certain features such as downloading Topics, changing your PIN, and more. </p><p><br></p><p>To see who your Administrator is, go to the Profile Chooser. The first profile listed and has "Administrator" written under their name is the Administrator. </p>]]></string>
>   <string name="faq_answer_5"><![CDATA[<p>If the Exploration Player is not loading</p><p><br></p><p>Check to see if the app is up to date:</p><p>- Go to the Play Store and make sure the app is updated to its latest version</p><p><br></p><p>Check your internet connection:</p><p>- If your internet connection is slow, try re-connecting to your Wi-Fi network or connecting to a different network.</p><p><br></p><p>Ask the Administrator to check their device and internet connection:</p><p>- Get the Administrator to troubleshoot using the steps above</p><p><br></p><p>Let us know if you still have issues with loading:</p><p>- Report a problem by contacting us at admin@oppia.org.</p>]]></string>
>   <string name="faq_answer_6"><![CDATA[<p>If your audio is not playing</p><p><br></p><p>Check to see if the app is up to date:</p><p>- Go to the Play Store and make sure the app is updated to its latest version</p><p><br></p><p>Check your internet connection:</p><p>- If your internet connection is slow, try re-connecting to your Wi-Fi network or connecting to a different network. Slow internet may cause the audio to load irregularly, making it difficult to play. </p><p><br></p><p>Ask the Administrator to check their device and internet connection:</p><p>- Get the Administrator to troubleshoot using the steps above</p><p><br></p><p>Let us know if you still have issues with loading:</p><p>- Report a problem by contacting us at admin@oppia.org.</p>]]></string>
>   <string name="faq_answer_7"><![CDATA[<p>To download an Exploration:</p><p>1. From the Home Page, tap on a Topic or Exploration.</p><p>2. From that Topic Page, tap on the Info tab.</p><p>3. Tap on Download Topic. </p><p>4. Depending on your app settings, you may need Administrator approval or stable Wifi connection to complete your download. If needed, once these requirements are satisfied, the Topic has been downloaded onto the device and can be used offline by all profiles. <p>]]></string>
>   <string name="faq_answer_8"><![CDATA[<p>If you cannot find your question or would like to report a bug, contact us at admin@oppia.org.</p>]]></string>
>   <string name="faq_answer_9"><![CDATA[<p>Oppia <i>"O-pee-yah"</i> (Finnish) - "to learn"</p><p><br></p><p>Oppia\'s mission is to help anyone learn anything they want in an effective and enjoyable way.</p><p><br></p><p>By creating a set of free, high-quality, demonstrably effective lessons with the help of educators from around the world, Oppia aims to provide students with quality education — regardless of where they are or what traditional resources they have access to.</p><p><br></p><p>As a student, you can begin your learning adventure by browsing the topics listed on the Home Page!</p>]]></string>
> </resources>
> ```
> 
> ```
> <?xml version="1.0" encoding="utf-8"?>
> <resources>
>   
>   <string-array name="faq_questions">
>     <item>@string/faq_question_1</item>
>     <item>@string/faq_question_2</item>
>     <item>@string/faq_question_3</item>
>     <item>@string/faq_question_4</item>
>     <item>@string/faq_question_5</item>
>     <item>@string/faq_question_6</item>
>     <item>@string/faq_question_7</item>
>     <item>@string/faq_question_8</item>
>     <item>@string/faq_question_9</item>
>   </string-array>
> 
>   <string-array name="faq_answers">
>     <item>@string/faq_answer_1</item>
>     <item>@string/faq_answer_2</item>
>     <item>@string/faq_answer_3</item>
>     <item>@string/faq_answer_4</item>
>     <item>@string/faq_answer_5</item>
>     <item>@string/faq_answer_6</item>
>     <item>@string/faq_answer_7</item>
>     <item>@string/faq_answer_8</item>
>     <item>@string/faq_answer_9</item>
>   </string-array>
> </resources>
> ```

@anandwana001 Actually, I don't have any idea what should I do after your previous comment.

**Comment by anandwana001** [comment]
> > @rt4914
> > 
> > ```
> > <?xml version="1.0" encoding="utf-8"?>
> > <resources>
> >   <string name="faq_question_1">How can I create a new profile?</string>
> >   <string name="faq_question_2">How can I delete a profile?</string>
> >   <string name="faq_question_3">How can I change my email/phone number?</string>
> >   <string name="faq_question_4">Who is an Administrator?</string>
> >   <string name="faq_question_5">Why is the Exploration player not loading?</string>
> >   <string name="faq_question_6">Why is my audio not playing?</string>
> >   <string name="faq_question_7">How do I download a Topic?</string>
> >   <string name="faq_question_8">I can\'t find my question here. What now?</string>
> >   <string name="faq_question_9">What is Oppia?</string>
> >   <string name="faq_answer_1"><![CDATA[<p>If it is your first time creating a profile and not have a PIN: </p> <p> 1. From the Profile Chooser, tap on <strong>Set up Multiple Profiles</strong>. </p> <p> 2. Create a PIN and <strong>Save</strong>. </p> <p> 3. Fill in all fields for the profile. </p> <ol> <li> (Optional) Upload a photo. </li> <li> Enter a name. </li> <li> (Optional) Assign a 3-digit PIN. </li> </ol> <p> 4. Tap <strong>Create</strong>. This profile is added to your Profile Chooser! <br/> <br/> If you have created a profile before and have a PIN: </p> <p> 1. From the Profile Chooser, tap on <strong>Add Profile</strong>. </p> <p> 2. Enter your PIN and tap <strong>Submit</strong>. </p> <p> 3. Fill in all fields for the profile. </p> <ol> <li> (Optional) Upload a photo. </li> <li> Enter a name. </li> <li> (Optional) Assign a 3-digit PIN. </li> </ol> <p> 4. Tap <strong>Create</strong>. This profile is added to your Profile Chooser! <br/> <br/> Note: Only the Administrator is able to manage profiles.</p>]]></string>
> >   <string name="faq_answer_2"><![CDATA[<p>Once a profile is deleted:</p> <p><br></p> <p>- The profile cannot be recovered.</p> <p>- Profile information such as name, photos, and progress will be permanently deleted.</p> <p><br></p> <p>To delete a profile (excluding the Administrator\'s):</p> <p>1. From the Administrator\'s Home Page, tap on the menu button on the top left.</p> <p>2. Tap on <strong>Administrator Controls</strong>.</p> <p>3. Tap on <strong>Edit Profiles</strong>.</p> <p>4. Tap on the Profile you would like to delete.</p> <p>5. At the bottom of the screen, tap <strong>Profile Deletion</strong>.</p> <p>6. Tap <strong>Delete</strong> to confirm deletion.</p> <p>Note: Only the Administrator is able to manage profiles.</p>]]></string>
> >   <string name="faq_answer_3"><![CDATA[<p>To change your email/phone number:</p> <p>1. From the Administrator\'s Home Page, tap on the menu button on the top left.</p> <p>2. Tap on <strong>Administrator Controls</strong>.</p> <p>3. Tap on <strong>Edit Account</strong>.</p> <p><br></p> <p>If you want to change your email:</p> <p>4. Enter your new email and tap <strong>Save</strong>.</p> <p>5. A confirmation link is sent to confirm your new email. The link will expire after days and must be clicked on to be associated with your account. </p> <p><br></p> <p>If changing your phone number:</p> <p>4. Enter your new phone number and tap <strong>Verify</strong>.</p> <p>5. A code is sent to confirm your new number. The code will expire after minutes and must be entered in the new screen to be associated with your account.</p>]]></string>
> >   <string name="faq_answer_4"><![CDATA[<p>An Administrator is the main user that manages profiles and settings for every profile on their account. They are most likely your parent, teacher, or guardian that created this profile for you. </p><p><br></p><p>Administrators have the ability to manage profiles, assign PINs, and change other settings under their account. Depending on your profile, Administrator permissions may be required for certain features such as downloading Topics, changing your PIN, and more. </p><p><br></p><p>To see who your Administrator is, go to the Profile Chooser. The first profile listed and has "Administrator" written under their name is the Administrator. </p>]]></string>
> >   <string name="faq_answer_5"><![CDATA[<p>If the Exploration Player is not loading</p><p><br></p><p>Check to see if the app is up to date:</p><p>- Go to the Play Store and make sure the app is updated to its latest version</p><p><br></p><p>Check your internet connection:</p><p>- If your internet connection is slow, try re-connecting to your Wi-Fi network or connecting to a different network.</p><p><br></p><p>Ask the Administrator to check their device and internet connection:</p><p>- Get the Administrator to troubleshoot using the steps above</p><p><br></p><p>Let us know if you still have issues with loading:</p><p>- Report a problem by contacting us at admin@oppia.org.</p>]]></string>
> >   <string name="faq_answer_6"><![CDATA[<p>If your audio is not playing</p><p><br></p><p>Check to see if the app is up to date:</p><p>- Go to the Play Store and make sure the app is updated to its latest version</p><p><br></p><p>Check your internet connection:</p><p>- If your internet connection is slow, try re-connecting to your Wi-Fi network or connecting to a different network. Slow internet may cause the audio to load irregularly, making it difficult to play. </p><p><br></p><p>Ask the Administrator to check their device and internet connection:</p><p>- Get the Administrator to troubleshoot using the steps above</p><p><br></p><p>Let us know if you still have issues with loading:</p><p>- Report a problem by contacting us at admin@oppia.org.</p>]]></string>
> >   <string name="faq_answer_7"><![CDATA[<p>To download an Exploration:</p><p>1. From the Home Page, tap on a Topic or Exploration.</p><p>2. From that Topic Page, tap on the Info tab.</p><p>3. Tap on Download Topic. </p><p>4. Depending on your app settings, you may need Administrator approval or stable Wifi connection to complete your download. If needed, once these requirements are satisfied, the Topic has been downloaded onto the device and can be used offline by all profiles. <p>]]></string>
> >   <string name="faq_answer_8"><![CDATA[<p>If you cannot find your question or would like to report a bug, contact us at admin@oppia.org.</p>]]></string>
> >   <string name="faq_answer_9"><![CDATA[<p>Oppia <i>"O-pee-yah"</i> (Finnish) - "to learn"</p><p><br></p><p>Oppia\'s mission is to help anyone learn anything they want in an effective and enjoyable way.</p><p><br></p><p>By creating a set of free, high-quality, demonstrably effective lessons with the help of educators from around the world, Oppia aims to provide students with quality education — regardless of where they are or what traditional resources they have access to.</p><p><br></p><p>As a student, you can begin your learning adventure by browsing the topics listed on the Home Page!</p>]]></string>
> > </resources>
> > ```
> > 
> > ```
> > <?xml version="1.0" encoding="utf-8"?>
> > <resources>
> >   
> >   <string-array name="faq_questions">
> >     <item>@string/faq_question_1</item>
> >     <item>@string/faq_question_2</item>
> >     <item>@string/faq_question_3</item>
> >     <item>@string/faq_question_4</item>
> >     <item>@string/faq_question_5</item>
> >     <item>@string/faq_question_6</item>
> >     <item>@string/faq_question_7</item>
> >     <item>@string/faq_question_8</item>
> >     <item>@string/faq_question_9</item>
> >   </string-array>
> > 
> >   <string-array name="faq_answers">
> >     <item>@string/faq_answer_1</item>
> >     <item>@string/faq_answer_2</item>
> >     <item>@string/faq_answer_3</item>
> >     <item>@string/faq_answer_4</item>
> >     <item>@string/faq_answer_5</item>
> >     <item>@string/faq_answer_6</item>
> >     <item>@string/faq_answer_7</item>
> >     <item>@string/faq_answer_8</item>
> >     <item>@string/faq_answer_9</item>
> >   </string-array>
> > </resources>
> > ```
> 
> @anandwana001 Actually, I don't have any idea what should I do after your previous comment. 

Actually, I was about to put some description. 

So, this is the html converted data we required for faq.

**Pull request closed without merging**